aea/decision_maker/default.py

```diff
@@ -19,22 +19,12 @@
 
 """This module contains the decision maker class."""
 
-import copy
-import logging
```

Comment:
Henceforth the default decision maker just accepts signing messages and processes them. There is no further checks.